data/Shashankzz0007.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Shashank Deshpande",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast! | DevOps | MERN Developer | Self-paced Full-Stack Developer",
+  "avatar": "https://github.com/Shashankzz0007.png",
+  "tags": ["Open Source", "javascript", "typescript", "Reactjs", "DevOps"],
+  "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/shashankz007" },
+    { "platform": "github", "url": "https://github.com/Shashankzz0007" },
+    {
+      "platform": "linkedin",
```

Comment:
Is LinkedIn permitted in the socials field? 🤔

Reply:
@Panquesito7, I think maybe it considers as social platform but if you are suggesting I will remove it.